FeedHenry/Sync/FHDefines.h

```diff
@@ -33,7 +33,8 @@ typedef NS_ENUM(NSInteger, FH_ACTION) {
 
 // cater for debug/release mode of logging
 #ifdef DEBUG
-#define DLog(...) NSLog(@"%s(%p) %@", __PRETTY_FUNCTION__, self, [NSString stringWithFormat:__VA_ARGS__])
+#define NSLog(FORMAT, ...) printf("%s %s\n", [[[NSDate date] description] UTF8String], [[NSString stringWithFormat:FORMAT, ##__VA_ARGS__] UTF8String]);
+#define DLog(...) NSLog(@"💜 %s %@", __PRETTY_FUNCTION__, [NSString stringWithFormat:__VA_ARGS__])
```

Comment:
I would suggest leaving a note here, so when we try to merge a new version of FH in, we know these are our changes. 
Also, instead of using printf consider using Apple's unified logging with `os_log` (https://developer.apple.com/documentation/os/logging)

Reply:
Maybe something like:
```swift
#define NSLog(FORMAT, ...) os_log("%{public}@", [[NSString stringWithFormat:FORMAT, ##__VA_ARGS__] UTF8String])
```